src/main/java/com/revature/controllers/QuestionEvalController.java

```diff
@@ -0,0 +1,62 @@
+package com.revature.controllers;
+
+import com.revature.domain.QuestionComment;
+import com.revature.domain.QuestionPool;
+import com.revature.services.QuestionLogic;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.*;
+import org.springframework.data.web.SortDefault.SortDefaults;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.data.web.SortDefault.SortDefaults'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)